frontend/src/themeSettings.ts

```diff
@@ -205,6 +205,15 @@ const theme = createTheme(
           },
         },
       },
+      MuiTooltip: {
+        styleOverrides: {
+          tooltip: {
+            maxWidth: '200px',
+            fontSize: '14px',
+            backgroundColor: 'rgba(78,78,78,0.9)',
```

Comment:
hex: #4E4E4E

Reply:
at this case i need to use rgba for adding oppacity only for background color. If i will write somethink like this: `backgroundColor: '#4E4E4E , opacity: 0.9', my text will be transparent too